### ISSUE TITLE
unsuccessful with azw3

### ISSUE THREAD

**Issue by Marlowe-007** [noDRM/DeDRM_tools#22]
### Question / bug report

Hello

I tried to do everything right:
- downloaded an azw3 from my Amazon's kindle page "download to transfer by USB"
- installed DeDRM plugin and costomized it with my Kindle's serial number
- imported azw3 in calibre => no problem
but when I want to open it says "this book is locked by DRM etc."

I should add this is a book I purchased for my Kindle.

I don't understand what I missed.
Note : I'm puzzled because I see in the log that it's talking about K4PC although I deinstalled it before doing all of the above.

I hope someone can help.

### Which version of Calibre are you running?

5.34

### Which version of the DeDRM plugin are you running?

10.0.2

### Log output

stdout+stderr from file dialog helper: [b'', b'']
piped data from file dialog helper: [b'+\x99\\6\xb6\x87\xcd\x8a\x18\x9d\x84\x8b\xaf\xe4m\x95\xa6\xbc\xc9w\x83aj\xee9\xb2\x83\x7f\xd6\xc7]\x87', b'C:\\Users\\AD\\Downloads\\Band 1&2 - Das gro\xc3\x9fe Buch der EAV_ Grundlagen und praktische Anwendungen (German Edition).azw3']
DeDRM v10.0.2: Trying to decrypt Band 1&2 - Das große Buch der EAV_ Grundlagen und praktische Anwendungen (German Edition).azw3
Using Library AlfCrypto DLL/DYLIB/SO
Using Library AlfCrypto DLL/DYLIB/SO
MobiDeDrm v1.0.
Copyright © 2008-2020 The Dark Reverser, Apprentice Harper et al.
Decrypting Kindle Format 8 ebook: Band 1&2 - Das große Buch der EAV: Grundlagen und praktische Anwendungen (German Edition)
Found 2 keys to try after 0.1 seconds
Crypto Type is: 2
DeDRM v10.0.2: Failed to decrypt with error: No key found in 2 PIDs tried.
DeDRM v10.0.2: Looking for new default Kindle Key after 0.1 seconds
searching for kinfoFiles in C:\Users\AD\AppData\Local
Found K4PC 1.25+ kinf2018 file: C:\Users\AD\AppData\Local\Amazon\Kindle\storage\.kinf2018
Decrypted key file using IDString '418257257' and UserName 'AD'
DeDRM v10.0.2: Found 1 new key
MobiDeDrm v1.0.
Copyright © 2008-2020 The Dark Reverser, Apprentice Harper et al.
Decrypting Kindle Format 8 ebook: Band 1&2 - Das große Buch der EAV: Grundlagen und praktische Anwendungen (German Edition)
Got DSN key from database default_key_1
Found 4 keys to try after 2.5 seconds
Crypto Type is: 2
DeDRM v10.0.2: Ultimately failed to decrypt after 2.5 seconds. Read the FAQs at Harper's repository: https://github.com/apprenticeharper/DeDRM_tools/blob/master/FAQs.md
Added Band 1&2 - Das große Buch der EAV: Grundlagen und praktische Anwendungen (German Edition) to db in: 0.1
Added 1 books in 3.0 seconds

**Comment by ElleKayEm** [comment]
When you were downloading, did you choose an E-ink Kindle (not a Fire) for the device?

(Since the serial number you had entered didn't work, it looked for Kindle for PC keys.  Even though the app was uninstalled, it looks like it left behind some files.)

**Comment by Marlowe-007** [comment]
Ok. Since you told me the problem was the serial number I deleted it and reentered it and relaunched everything... and it WORKS!
This is weird though because the first time around I rechecked many times the SN was right, and I did exactly the same steps this time.
But anyway, this is great. I need to have an Epub version so that my Text to speech app can read the book (I have eyesight issues and the Kindle TTS is terrible).

Thanks a LOT!
